dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ConfigurationRestRepository.java

```diff
@@ -72,6 +75,31 @@ public Page<PropertyRest> findAll(Context context, Pageable pageable) {
         throw new RepositoryMethodNotImplementedException("No implementation found; Method not allowed", "");
     }
 
+    /**
+     * Gets the value of all configuration properties exposed via REST
+     * @param pageable pagination information
+     * @return all configuration properties as an array
+     */
+    @SearchRestMethod(name = "exposed")
```

Comment:
I've just left a comment on the RestContract. If you consider it, then this name should also be changed.

Reply:
actually exposed is more close to other existing search method name (top for communities) but I'm ok with both isExposed than exposed as long as the rest implementation and the rest contract match